components/modal/subscriptions-modal/subscriptions-preview/table/styles.scss

```diff
@@ -0,0 +1,9 @@
+.c-subscription-preview-table {
+  .disclaimer-container {
+    width: 100%;
+    background-color: rgba(250,183,46, 0.1);
+    border-radius: 4px;
+    padding: 10px;
```

Comment:
Let's try to use the `$space` variable instead and keep consistency with spacing.